test/easyblocks/module.py

```diff
@@ -90,6 +90,59 @@ def setUp(self):
         fd, self.eb_file = tempfile.mkstemp(prefix='easyblocks_module_only_test_', suffix='.eb')
         os.close(fd)
 
+    def test_make_module_pythonpackage(self):
+        """Test make_module_step of PythonPackage easyblock."""
+        app_class = get_easyblock_class('PythonPackage')
+        self.writeEC('PythonPackage', name='testpypkg', version='3.14')
+        app = app_class(EasyConfig(self.eb_file))
+
+        # install dir should not be there yet
+        self.assertFalse(os.path.exists(app.installdir))
+
+        # create install dir and populate it with subdirs/files
+        mkdir(app.installdir, parents=True)
+        # $PATH, $LD_LIBRARY_PATH, $LIBRARY_PATH, $CPATH, $PKG_CONFIG_PATH
+        write_file(os.path.join(app.installdir, 'bin', 'foo'), 'echo foo!')
+        write_file(os.path.join(app.installdir, 'include', 'foo.h'), 'bar')
+        write_file(os.path.join(app.installdir, 'lib', 'libfoo.a'), 'libfoo')
+        write_file(os.path.join(app.installdir, 'lib', 'python2.7', 'site-packages', 'foo.egg'), 'foo egg')
+        write_file(os.path.join(app.installdir, 'lib64', 'pkgconfig', 'foo.pc'), 'libfoo: foo')
+
+        # create module file
+        app.make_module_step()
+
+        self.assertTrue(TMPDIR in app.installdir)
+        self.assertTrue(TMPDIR in app.installdir_mod)
+
+        modtxt = None
+        for cand_mod_filename in ['3.14', '3.14.lua']:
+            full_modpath = os.path.join(app.installdir_mod, 'testpypkg', cand_mod_filename)
+            if os.path.exists(full_modpath):
+                modtxt = read_file(full_modpath)
+                break
```

Comment:
that's not supported yet; and we don't really care here, that's not what we care about in this test?